src/ainft.ts

```diff
@@ -35,7 +41,7 @@ export default class AinftJs {
   public textToArt: TextToArt;
   public activity: Activity;
   public eth: Eth;
-  public ai: BaseAi;
+  public ai: Ai;
```

Comment:
(question) it looks like ChatAi does not inherit from Ai nor call any call any methods of Ai. can ChatAi be added as a member of AinftJs class instead discarding Ai class?

Reply:
Done, thanks!